### DISCUSSION TITLE
Waypoints for NPC's

### DISCUSSION THREAD

**Discussion by javirocha** [replit/kaboom#143] [Q&A]
Is there a way to create waypoints in which NPC's can patrol? I've tried randomizing the direction on collision with no success. I Would like to create a sort of maze in which foes can patrol doing certain routes or random wandering. Thanks in advance, I'm really enjoying this so far.

**Comment by lajbel** [comment]
You can use some function of where you want them to move, this would go perfectly with the moveTo () function that I proposed a few days ago #134
